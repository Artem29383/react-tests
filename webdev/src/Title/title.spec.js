import Title from "./Title";
import React from "react";

describe('Title component', () => {
  it('render component title with props', () => {
    const component = shallow(<Title title='Test props' />)
    expect(component).toMatchSnapshot();
  })

  it('render component title without props', () => {
    const component = shallow(<Title />)
    expect(component).toMatchSnapshot();
  })
})