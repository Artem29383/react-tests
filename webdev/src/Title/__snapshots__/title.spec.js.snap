// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component render component title with props 1`] = `
<h1
  className="title"
>
  Test props
</h1>
`;

exports[`Title component render component title without props 1`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
